docs/source/user_guide/configuration_options.rst

```diff
@@ -0,0 +1,890 @@
+==============================
+Receptor Configuration Options
+==============================
+
+---------------------
+Receptor command line
+---------------------
+
+Command line arguments have the form: receptor [--<action> [<param>=<value> ...] ...]
+
+<action> are listed below.  When an action has parameters, they are listed in their own section.
+
+-------------
+Miscellaneous
+-------------
+
+.. list-table:: Miscellaneous
+    :header-rows: 1
+    :widths: auto
+
+    * - Action
+      - Description
+    * - --bash-completion
+      - Generate a completion script for the bash shell
+    * - --config <filename>
+      - Load additional config options from a YAML file
+    * - --control-service
+      - Run a control service
```

Comment:
```suggestion
      - Runs a control service.
```

Reply:
Is the control service a special type of node? Should we say "control service node"?